cmd/attack/jvm.go

```diff
@@ -150,6 +151,26 @@ func NewJVMRuleFileCommand(dep fx.Option, options *core.JVMCommand) *cobra.Comma
 	return cmd
 }
 
+func NewJVMMySQLCommand(dep fx.Option, options *core.JVMCommand) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "mysql [options]",
+		Short: "inject fault into MySQL client",
+		Run: func(*cobra.Command, []string) {
+			options.Action = core.JVMMySQLAction
+			utils.FxNewAppWithoutLog(dep, fx.Invoke(jvmCommandFunc)).Run()
+		},
+	}
+
+	cmd.Flags().StringVarP(&options.SQLType, "sql-type", "", "", "the match sql type")
+	cmd.Flags().StringVarP(&options.Database, "database", "d", "", "the match database")
```

Comment:
If this field is required, we can add the required information on help descriptipn.

Reply:
ditto, I will update the description and comment